### DISCUSSION TITLE
Bad font rendering on mirth connect administrator

### DISCUSSION THREAD

**Discussion by firestarr28** [nextgenhealthcare/connect#4541] [General]
Hello everybody
I installed mirth administrator on Gnu / Linux Kde Neon but the characters in the raw message, transformed, etc are horribly visible!
![Screenshot_20210513_104955](https://user-images.githubusercontent.com/11976361/118103009-ba4fd000-b3d9-11eb-88f4-58eb3532054e.png)

**Comment by jonbartels** [comment]
Does installing Tahoma help? A related issue is https://github.com/nextgenhealthcare/connect/issues/2582 searching the fourms or issues for "tahoma" or "font" may help. I recall this being a common issue but I could not readily locate a specific solution.

**Reply by tonygermano** [reply]
Definitely looks like a font replacement issue. I'm thinking that window might use a different font than Tahoma, though, which is used for tables and menus.

**Comment by firestarr28** [comment]
Hello
i tried but unfortunately it didn't help :(
thanks anyway
